Packs/TrustwaveFusion/Integrations/TrustwaveFusion/TrustwaveFusion.py

```diff
@@ -0,0 +1,587 @@
+            "sortField": "createdOn",
+            "sortDescending": "false",
+        }
+        if ticket_types:
+            params["type"] = ",".join(ticket_types)
+        demisto.debug(f"params: {params}")
+        tickets = client.search_tickets(**params)
+        latest_timestamp = search_since
+        if tickets:
+            demisto.debug(f"Found {len(tickets)} tickets from Fusion")
+            for tkt in tickets:
+                incident = {
+                    "name": tkt["subject"],
+                    "occurred": tkt["createdOn"],
+                    "description": tkt.get("description", ""),
+                    "severity": SEVERITY_MAP.get(tkt.get("priority"), 0),
+                    "rawJSON": json.dumps(tkt),
+                }
+                dt = dateparser.parse(tkt["createdOn"], settings={"TIMEZONE": "UTC"})
+                latest_timestamp = max(dt.timestamp(), latest_timestamp)
+                incidents.append(incident)
+    except DemistoException as e:
+        demisto.error(f"ERROR: {e}")
+        demisto.error(e.res)
+
+    # One second in the future to prevent duplicates
+    last_run = {"last_fetch": latest_timestamp + 1}
+    demisto.setLastRun(last_run)
+    demisto.incidents(incidents)
+
+
+def get_ticket_command(client, args):
+    ticket = client.get_ticket(args.get("id"))
+
+    readable = [
+        "| field | value |",
+        "|-|-|",
+    ]
+    for k, v in ticket.items():
+        str_value = stringEscapeMD(formatCell(v), True, True)
+        if len(str_value) > 200:
+            str_value = str_value[:200] + "....[Trucated]"
+        readable.append(f"| {k} | {str_value} |")
+
+    command_results = CommandResults(
+        outputs_prefix="Trustwave.Ticket",
+        outputs_key_field="number",
+        outputs=ticket,
+        readable_output="\n".join(readable),
+    )
+    return command_results
+
+
+def add_ticket_comment_command(client, args):
+    id = args.get("id")
+    comment = args.get("comment")
+    client.add_ticket_comment(id, comment)
+
+    return "Success"
+
+
+def close_ticket_command(client, args):
+    id = args.get("id")
+    comment = args.get("comment")
+    client.add_ticket_comment(id, comment)
+
+    return "Success"
+
+
+def get_finding_command(client, args):
+    finding = client.get_finding(args.get("id"))
+
+    # TODO: Add readable_output (markdown) for warroom view
+    command_results = CommandResults(
+        outputs_prefix="Trustwave.Finding", outputs_key_field="id", outputs=finding
+    )
+    return command_results
+
+
+def get_asset_command(client, args):
+    asset = client.get_asset(args.get("id"))
+
+    # TODO: Add readable_output (markdown) for warroom view
+    command_results = CommandResults(
+        outputs_prefix="Trustwave.Asset", outputs_key_field="id", outputs=asset
+    )
+    return command_results
+
+
+def get_updated_tickets_command(client, args):
+    updated_since = arg_to_datestring(args.get("since"),
```

Comment:
if there's non returned, then please return a readable output that states that and not an error.